xrmocap/core/evaluation/mvp_evaluation.py

```diff
@@ -54,14 +277,12 @@ def evaluate_map(self, pred_kps3d: torch.Tensor) \
             if len(gt_frame_kps3d) == 0:
                 continue
 
-            pred_frame_kps3d = pred_kps3d[index].copy()
-            pred_frame_kps3d_valid = pred_frame_kps3d[pred_frame_kps3d[:, 0,
-                                                                       3] >= 0]
+            pred_frame_kps3d_valid = pred_kps3d[frame_idx_all_scenes].copy()
 
             for pred_person_kps3d in pred_frame_kps3d_valid:
                 mpjpes = []
                 for gt_person_kps3d in gt_frame_kps3d:
-                    vis = gt_person_kps3d[:, -1] > self.kps_thr
+                    vis = gt_person_kps3d[:, -1] > threshold
 
                     mpjpe = np.mean(
```

Comment:
Can we calc mpjpe or others from [metrics.py](https://github.com/openxrlab/xrmocap/blob/main/xrmocap/core/evaluation/metrics.py)?

Reply:
Replaced the PCP and MPJPE calculation for Shelf and Campus with metrics.py functions in [8696023](https://github.com/openxrlab/xrmocap/pull/47/commits/8696023dc3de99fcd178d41f67a14d8b7250cd8c).